cmake/SetupCodeChecks.cmake

```diff
@@ -509,9 +509,10 @@ macro(blt_add_astyle_target)
     if(BLT_REQUIRED_ASTYLE_VERSION)
         # The user may only specify a part of the version (e.g. just the maj ver)
         # so check for substring
-        string(FIND ${_astyle_version} ${BLT_REQUIRED_ASTYLE_VERSION} VERSION_POS)
+        string(FIND "${_astyle_version}" ${BLT_REQUIRED_ASTYLE_VERSION} VERSION_POS)
```

Comment:
This was an empty string which helpfully causes a CMake parsing error.

Reply:
Good catch! 

Does that mean that astyle was not emitting a version number? If so, was it an old version or a new version of astyle that was causing the issues?

Reply:
This came up when Serac was built in WSL with an autodetected clang-format that returned an empty string (i guess).  I dont have access to the system.  So I'm guarding all future places this could happen. Then doing the chain of PRs so they can be unblocked.

Reply:
Got it -- thanks for the context.